README.md

```diff
@@ -31,6 +31,11 @@ Documentation for the API was created in OpenAPI format and is hosted [here](htt
 
 - add better error handling when dealing with response data (display error message when something will fail)
 - add caching to avoid server querying with every location change (if location was visited then keep it in memory and use it again)
+- add instruction how to run project locally with Docker
```

Comment:
It's already in the Wiki page, ref. https://github.com/rBiblia/rbiblia-web/wiki/How-to-start%3F